cmd/containerd-shim-runhcs-v1/serve.go

```diff
@@ -265,34 +265,20 @@ func setupDumpStacks() {
 	go func() {
 		for {
 			windows.WaitForSingleObject(handle, windows.INFINITE)
-			dumpStacks(true)
+			dumpStacks()
 		}
 	}()
 	return
 }
 
-func dumpStacks(writeToFile bool) {
-	var (
-		buf       []byte
-		stackSize int
-	)
-	bufferLen := 16384
-	for stackSize == len(buf) {
-		buf = make([]byte, bufferLen)
-		stackSize = runtime.Stack(buf, true)
-		bufferLen *= 2
+func dumpStacks() {
```

Comment:
Lets put this code in the etw callback.

Reply:
And lets delete `setupDumpStacks`. `shimdiag` is the supported way to do this now